src/components/SettingsDialog/Appearance/Appearance.tsx

```diff
@@ -23,34 +35,38 @@ export const Appearance = () => {
         <h2 className="settings-dialog__header-text">{t("SettingsDialog.Appearance")}</h2>
       </header>
       <div className="appearance-container">
-        {/* TODO: turn component to html select component for better accessibility */}
-        <SettingsButton
-          className="appearance-settings_language-dropdown"
-          label={t("Appearance.colorScheme")}
-          aria-haspopup="listbox"
-          aria-expanded={showThemeDropdown}
-          onClick={() => setShowThemeDropdown((o) => !o)}
-          onBlur={(e) => handleBlur(e, () => setShowThemeDropdown(false))}
-        >
-          <ThemeSettingsDropdown showDropdown={showThemeDropdown} setShowDropdown={setShowThemeDropdown} />
-        </SettingsButton>
-        {/* <SettingsButton */}
-        {/*   className="appearance-settings_sync-button" */}
-        {/*   onClick={() => { */}
-        {/*     setSyncMode((prevVal) => !prevVal); */}
-        {/*   }} */}
-        {/* > */}
-        {/*   <div className="appearance-settings_sync-button_label"> */}
-        {/*     <p>{t("Appearance.SyncMode")}</p> */}
-        {/*     <p>{t("Appearance.SyncModeDescription")}</p> */}
-        {/*   </div> */}
-        {/*   <SettingsToggle active={syncMode} /> */}
-        {/* </SettingsButton> */}
-        {/* <SettingsButton  */}
-        {/*   className="appearance-settings_notifications-button"  */}
-        {/*   label={t("Appearance.AllowNotifications")} onClick={() => setShowNotifications(!showNotifications)}> */}
-        {/*   <SettingsToggle active={showNotifications} /> */}
-        {/* </SettingsButton> */}
+        <div className="appearance-settings__theme-container">
+          <span className="appearance-settings__theme-title">{t("Appearance.colorScheme")}</span>
+          <div className="appearance-settings__theme-options">
+            <label htmlFor="auto" className="appearence-settings__theme-option" title={t("Appearance.SyncModeDescription")}>
+              <input id="auto" type="radio" value="auto" name="theme" checked={theme === "auto"} onChange={() => setTheme("auto")} />
+              <div className="appearance-settings__auto-preview">
+                <img src={ThemePreviewLight} alt={`${t("Appearance.colorScheme")} Auto`} />
+                <img src={ThemePreviewDark} alt={`${t("Appearance.colorScheme")} Auto`} />
+              </div>
+              <span>
+                <AutoIcon className="appearance-settings__theme-icon" />
+                Auto
+              </span>
+            </label>
+            <label htmlFor="light" className="appearence-settings__theme-option">
+              <input id="light" type="radio" value="light" name="theme" checked={theme === "light"} onChange={() => setTheme("light")} />
+              <img src={ThemePreviewLight} alt={`${t("Appearance.colorScheme")} ${t("Appearance.colorSchemeLight")}`} />
```

Comment:
Shouldn't have the alt text also `Appearance.colorSchemeLight` as translation instead of only `colorScheme`?

Reply:
see above